apps/aecore/src/aec_peers.erl

```diff
@@ -505,6 +505,8 @@ ping_peer(Peer) ->
                 {error, protocol_violation} ->
                     block_peer(Uri);
                 _ ->
```

Comment:
I see this is equivalent to `{error, _}`.

Reply:
No it is not, because in protocol validation the side effect is blocking the peer, for other errors (like difference in version) the peer is not blocked.